### PR TITLE
Add storage limit example 

### PR DESCRIPTION
**Pull request by markturansky**
Added example for applying LimitRange and ResourceQuota to storage.

Also updated the ResourceQuota docs to include storage.

@derekwaynecarr

---

This change is [Reviewable](https://reviewable.io/reviews/kubernetes/kubernetes.github.io/1089)

### CODE REVIEW

**Comment by markturansky** [comment]
@abhgupta

**Comment by jaredbhatti** [comment]
Can you rebase?

**Comment by markturansky** [comment]
@jaredbhatti rebased

**Comment by devin-donnelly** [comment]
Docs LGTM from me. Pending rebase and tech LGTM.

**Comment by markturansky** [comment]
Rebased again.

File: docs/admin/resourcequota/index.md [hunk 1]
```diff
@@ -63,6 +63,7 @@ in a namespace can be limited.  The following compute resource types are support
 | `memory` | Across all pods in a non-terminal state, the sum of memory requests cannot exceed this value. |
 | `requests.cpu` | Across all pods in a non-terminal state, the sum of CPU requests cannot exceed this value. |
 | `requests.memory` | Across all pods in a non-terminal state, the sum of memory requests cannot exceed this value. |
+| `requests.storage` | Across all persistent volume claims, the sum of storage requests cannot exceed this value. |
```
Comment (derekwaynecarr) [inline docs/admin/resourcequota/index.md]: do you mind splitting this into a separate PR?  this made kube 1.4, but the rest will not make kube 1.4 given feature freeze.

**Comment by derekwaynecarr** [comment]
The content is LGTM, but I don't think the LimitRange support will make 1.4 since we are past feature freeze.  Can you split the requests.storage piece into a PR for 1.4, and leave this open for 1.5?

**Comment by devin-donnelly** [comment]
Is this change only relevant after the 1.4 release? If that's the case, please change the target branch to release-1.4.

**Comment by markturansky** [comment]
This PR is for 1.5 after https://github.com/kubernetes/kubernetes/pull/30145 is merged.

https://github.com/kubernetes/kubernetes.github.io/pull/1207 was created for the PVC quota that made it into 1.4.

**Comment by devin-donnelly** [comment]
If this isn't relevant until 1.5, it's going to be waiting around for a while. When we cut a release-1.5 branch, I recommend retargeting this PR there.

**Comment by derekwaynecarr** [comment]
LGTM, looks like the branch is out of date now though.

**Comment by jaredbhatti** [comment]
Can you retarget this against the 1.5 branch?

**Comment by markturansky** [comment]
@jaredbhatti why am I targeting the 1.5 branch? This PR got its own merge into 1.4 but not master, so now I have to make a 1.5 branch PR? If we merge into master, won't it make it into all future branches ... ?

**Comment by jaredbhatti** [comment]
Ah, yes. You're right @markturansky.  Just rebase and I'll merge.

**Comment by markturansky** [comment]
@jaredbhatti rebased

**Comment by jaredbhatti** [comment]
Erg.  Sorry to ask this again, but can you rebase?

**Comment by devin-donnelly** [comment]
We need to resolve the merge conflict in the Guides ToC before we can merge.

**Comment by jaredbhatti** [comment]
Ping.  Can you fix the conflict?

**Comment by jaredbhatti** [comment]
Last ping :(

**Comment by markturansky** [comment]
@jaredbhatti  thanks for your patience. Rebased and updated the title of the doc to show better in the new menu layout.
